app/src/main/res/values/strings.xml

```diff
@@ -181,10 +181,13 @@ be temporary, and you can try again later.</li>
     <string name="turbo_mode_disabled_toast">Turbo Mode disabled</string>
     <string name="onboarding_turbo_mode_title">Turbo Mode helps you browse faster</string>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
+    <string name="onboarding_turbo_mode_body2">Turbo Mode automatically blocks trackers and ads to help get you where you’re going faster. If there’s a speed bump – like a site that isn’t behaving the way you expect – just turn Turbo Mode off.</string>
     <!-- Button text for keeping Turbo Mode enabled in onboarding -->
     <string name="button_turbo_mode_keep_enabled">Keep enabled</string>
+    <string name="button_turbo_mode_keep_enabled2">Keep Turbo Mode enabled</string>
     <!-- Button text for turning off Turbo Mode in onboarding -->
     <string name="button_turbo_mode_turn_off">Turn off</string>
+    <string name="button_turbo_mode_turn_off2">Turbo Turbo Mode off</string>
```

Comment:
Should this say '_Turn_ Turbo Mode off'?

Reply:
Good catch